test/e2e/utils/test_context.go

```diff
@@ -35,15 +35,16 @@ const prometheusOperatorURL = "https://raw.githubusercontent.com/coreos/promethe
 // TestContext specified to run e2e tests
 type TestContext struct {
 	*CmdContext
-	TestSuffix string
-	Domain     string
-	Group      string
-	Version    string
-	Kind       string
-	Resources  string
-	ImageName  string
-	Kubectl    *Kubectl
-	BinaryName string
+	TestSuffix    string
+	Domain        string
+	Group         string
+	Version       string
+	SecondVersion string
```

Comment:
IMO we sould not add Secondversion here. What does it means for the TestContext?

Reply:
Sorry, this was an intermediate change while testing. Have reverted it back.